superbit_lensing/pipe.py

```diff
@@ -394,6 +394,7 @@ def _setup_run_command(self, run_options):
         base = f'python {filepath} {mock_dir} {outfile}'
 
         options = self._setup_options(run_options)
+        #if
```

Comment:
maybe remove this :)

Reply:
Sorry, I don't understand: why would we remove the `_req_fields`?

Reply:
I just meant the `#if`!

Reply:
Done